deploy/marlowe-runtime/templates/marlowe-proxy.yaml

```diff
@@ -58,6 +58,10 @@ spec:
         properties:
           annotations:
             "external-dns.alpha.kubernetes.io/hostname": "marlowe-runtime-{{ . }}.{{ $instance.parentDomain }}"
+            "service.beta.kubernetes.io/aws-load-balancer-name": "marlowe-runtime-{{ . }}-{{ $instanceName }}"
+            "service.beta.kubernetes.io/aws-load-balancer-nlb-target-type": "instance"
+            "service.beta.kubernetes.io/aws-load-balancer-scheme": "internet-facing"
+            "service.beta.kubernetes.io/aws-load-balancer-type": "external"
```

Comment:
Can you explain how the LoadBalancer in front of marlowe-proxy works? I just want to make sure we're not causing any unforeseen problems